app/models/response_node.rb

```diff
@@ -28,4 +32,15 @@ def debug_tree(indent: 0)
   def casted_value
     nil
   end
+
+  #_destroy defaults to false unless set otherwise
+  def _destroy
+    @destroy.nil? ? false : @destroy
+  end
+
+  # A flag indicating whether this node should be destroyed before save.
+  # convert string 'true'/'false' to boolean
+  def _destroy=(d)
+    @destroy = d.is_a?(String) ? d == "true" : d
```

Comment:
Where are we taking care of deleting the node if relevant is false?

Reply:
That comes in 8663, integrating the controller and actually saving the response.